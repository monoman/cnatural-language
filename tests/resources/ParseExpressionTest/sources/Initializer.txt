contacts = new List<Contact> {
	new Contact {
		Name = "Chris Smith",
		PhoneNumbers = { "206-555-0101", "425-882-8080" }
	},
	new Contact {
		Name = "Bob Harris",
		PhoneNumbers = { "650-555-0199" }
	}
}
