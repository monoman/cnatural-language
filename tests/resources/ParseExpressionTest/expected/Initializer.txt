Assign(Name(contacts) Assign new List<Contact>{{new Contact{Name=Literal(String, ["Chris Smith"]), PhoneNumbers={{Literal(String, ["206-555-0101"])}, {Literal(String, ["425-882-8080"])}}}}, {new Contact{Name=Literal(String, ["Bob Harris"]), PhoneNumbers={{Literal(String, ["650-555-0199"])}}}}})